blog/Webots-2018-b-release.md

```diff
@@ -0,0 +1,127 @@
+
+## WREN: Webots Renderer
+
+For the past decade, Webots has used the [Object-Oriented Graphics Rendering Engine](https://www.ogre3d.org/) (OGRE) to handle 3D rendering. 
+We encountered many roadblocks with OGRE, especially in modernizing our worlds and achieving a greater level of realism, and as such decided that it was better for us to implement our own bespoke rendering engine - from scratch!
+This allows us to implement and use only the features we need / desire, giving us a very lightweight and efficient renderer.
+
+
+We've spent the past year doing just this, and are proud to present our own bespoke OpenGL 3.3 based renderering engine, WREN (Webots RENderer).
+Thus, Webots R2018b and all future versions of Webots will make use of WREN for all 3D work.
+
+### Lightning Fast
+
+As a result of implementing only what we need, and focusing on using modern techniques introduced with OpenGL 3+, Webots with WREN is much faster than with OGRE.
+OGRE 1.9 (the version we used) is a CPU-bound renderer, which is bad news for Webots, which is optimised for single-thread performance.
+Ideally, a 3D renderer for use with Webots should be GPU bound, and be as CPU-overhead minimal as possible, to provide "room" for the simulation's CPU-sided simulation workload (robot controllers, physics, GUI) as possible.
+WREN is extremely CPU-light, and is GPU-bound for rendering.
+This has allowed us to improve world performance considerably.
+
+
+To illustrate this in practical terms, the simulation speedometer in "Run" mode is used as our performance indicator, to show the real-life implications of faster rendering in a Webots simulation.
+The following benchmarks were performed on Windows 10, 64-bit with an Intel 6700k processor @ 3.40GHz and an NVIDIA GTX 970 graphics cards on some familiar webots worlds.
+
+|    World    | R2018a | R2018b | speedup |
+|:-----------:|:------:|:------:|:-------:|
+|   city.wbt  |  2.82x |  7.60x |  2.70x  |
+| battery.wbt |  35.1x |  43.6x |  1.24x  |
+| create.wbt  | 30.4x  |  52.4x |  1.72x  |
+
+
+As you can see from these tables, even with robot controller taking a large part of the timestep in these worlds, all of them run considerably faster than before, with a greater speedup in complex worlds due to higher rendering demand. 
+WREN now shines in large worlds, and is still way ahead of OGRE even in simple scenarios.
+
+### Physically-Based Rendering
+
+%figure "A model of a telephone, from the GLTF sample library, rendered in Webots."
+![telephone](images/telephone.png)
+%end
+
+In a bid to make Webots look as realistic as possible, we have implemented a Physically-Based Rendering pipeline into Webots.
+Physically based rendering relies on improved lighting models, and the use of several additional non-colour textures (normal maps, roughness maps and metalness maps) to represent a material's interactions with light to a very high degree of realism.
+This is useable with the new PBRAppearance and Cubemap nodes.
```

Comment:
As a general comment, we should link to the corresponding doc (e.g. here to `PBRAppearance` and `Cubemap` nodes in reference manual).